Hudl.FFmpeg.Core/Command/BaseTypes/ICommand.cs

```diff
@@ -12,7 +12,11 @@ public interface ICommand
             where TProcessorType : class, ICommandProcessor, new()
             where TBuilderType : class, ICommandBuilder, new();
 
-        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(TProcessorType commandProcessor)
+        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(int? timeoutMilliseconds)
+            where TProcessorType : class, ICommandProcessor, new()
+            where TBuilderType : class, ICommandBuilder, new();
+
+        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(TProcessorType commandProcessor, int? timeoutMilliseconds)
```

Comment:
How concerned are we with breaking changes in this project? I see this and a couple others, which normally means this should be a major version increase.